### ISSUE TITLE
ssh-agent support

### ISSUE THREAD

**Issue by dee-kryvenko** [plumber-cd/runtainer#11]

**Comment by dee-kryvenko** [comment]
https://github.com/FutureAdvisor/pinata-ssh-agent

**Comment by dee-kryvenko** [comment]
Also pgp agent

**Comment by dee-kryvenko** [comment]
`SSH_AUTH_SOCK` forwarding has been implemented. I was, however, unable to test and enjoy this feature myself. While I was able to test that the socket file being mounted in principal and env variable been injected - I am running Rancher Desktop at the moment, and unix domain sockets are not currently supported to be forwarded to the Lima VM (due to that its using sshfs I guess). Maybe it will change with them switching to 9p, who knows. Parent issue: https://github.com/lima-vm/lima/issues/834

Still, maybe somebody going to find this feature useful. I myself decided to give up and go alternative approach: https://github.com/plumber-cd/runtainer/issues/23. This way I will be able to securely store credentials in my local k8s cluster and make them available inside my containers.
